lib/Resque/Redis.php

```diff
@@ -228,7 +232,7 @@ public function __call($name, $args)
 			return $this->driver->__call($name, $args);
 		}
 		catch (CredisException $e) {
-			return false;
+			throw new Resque_RedisException($e);
```

Comment:
Logging (and other) libraries can/will provide more context/information if the exception is passed as the third argument to `\Exception` where [`Exception::getPrevious`](http://php.net/manual/en/exception.getprevious.php) can be used.

For example

```php
throw new Resque_RedisException('Some message', 0, $e)
```

In this case $e will be cast to a string and we will lose some of that additional reporting.